[2024-10-18T13:30:27.338+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-18T13:30:27.429+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_2nd_task scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2024-10-18T13:30:27.457+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_2nd_task scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2024-10-18T13:30:27.459+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-18T13:30:27.562+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): this_is_2nd_task> on 2023-03-05 00:00:00+00:00
[2024-10-18T13:30:27.609+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'this_is_2nd_task', 'scheduled__2023-03-05T00:00:00+00:00', '--job-id', '4261', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp0c6ywrgx']
[2024-10-18T13:30:27.629+0000] {standard_task_runner.py:105} INFO - Job 4261: Subtask this_is_2nd_task
[2024-10-18T13:30:27.628+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12313) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-18T13:30:27.637+0000] {standard_task_runner.py:72} INFO - Started process 12329 to run task
[2024-10-18T13:30:27.798+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v4.this_is_2nd_task scheduled__2023-03-05T00:00:00+00:00 [running]> on host ad4831ffc3d7
[2024-10-18T13:30:28.122+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sourabh' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='this_is_2nd_task' AIRFLOW_CTX_EXECUTION_DATE='2023-03-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-05T00:00:00+00:00'
[2024-10-18T13:30:28.130+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-18T13:30:28.222+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-18T13:30:28.228+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is my second task and will run after task one ']
[2024-10-18T13:30:28.264+0000] {subprocess.py:86} INFO - Output:
[2024-10-18T13:30:28.272+0000] {subprocess.py:93} INFO - Hello world, this is my second task and will run after task one
[2024-10-18T13:30:28.273+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-18T13:30:28.353+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-18T13:30:28.355+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=this_is_2nd_task, run_id=scheduled__2023-03-05T00:00:00+00:00, execution_date=20230305T000000, start_date=20241018T133027, end_date=20241018T133028
[2024-10-18T13:30:28.418+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-18T13:30:28.498+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
